tox-uv is not needed to run tests in our environment.

--- build-1.5.0/tox.toml.orig
+++ build-1.5.0/tox.toml
@@ -1,4 +1,3 @@
-requires = ["tox-uv>=1.28"]
 env_list = [
     "fix",
     "type",
